vercel/src/ascii-preview/Index.svelte

```diff
@@ -0,0 +1,51 @@
+<script lang="ts">
+  import Nav from '../lib/Nav.svelte'
+  import AsciiPreview from '../lib/AsciiPreview.svelte';
+  import type { Languages } from '../types';
+  import  getLanguages  from '../get-languages';
+
+  const languages = (async () => {
+    const languageObject = await getLanguages();
+    // HACK This looks unnecessary to destructure and restructure, but Svelte
+    // was freaking out about accessing colors.property in the template.
+    return Object.entries(languageObject).map(([name, { ascii, colors }]) => ({
+      name,
+      ascii,
+      ...colors,
+    }))
+    .sort((a, b) => a.name.localeCompare(b.name));
+  })();
+</script>
+
+<main>
+  <h1>ASCII Preview</h1>
+
+  <div class="card">
+    <Nav active="/ascii-preview" />
+  </div>
+  <h2>Languages</h2>
+  {#await languages}
+    <p>Fetching languages...</p>
+  {:then result}
```

Comment:
```suggestion
  {#await languages}
    <p>Fetching languages...</p>
  {:then result}
    <h2>Languages ({result.length})</h2>
```

In order to display the number of languages

Reply:
What do you think about `{{ result.length }} Languages`? Think it's bit more like plain English.

Reply:
Yes, `{result.length } languages` may be more readable

Reply:
Oh, sorry, I completely misread your code suggestion and saw it as a Vue-like `{{ value }}`. I didn't realize there were parentheses 😆